fsm/join_sync_dnskeys.go

```diff
@@ -28,97 +28,6 @@ var FsmJoinSyncDnskeys = music.FSMTransition{
 //      for the next action? I think so. I.e. this implementation (VerifyDnskeysSynched) is
 //      extremely similar to the JoinAddCdsPreCondition function that is the PreCondition for
 //      the next step (adding CDS/CDNSKEYs).
-func VerifyDnskeysSynched(z *music.Zone) bool {
```

Comment:
The entire function has just been moved down under the action, no changes were intentionally made to the code